plugins/LilyPad/resource.h

```diff
@@ -146,7 +147,7 @@
 // 
 #ifdef APSTUDIO_INVOKED
 #ifndef APSTUDIO_READONLY_SYMBOLS
-#define _APS_NEXT_RESOURCE_VALUE        112
+#define _APS_NEXT_RESOURCE_VALUE        114
```

Comment:
why not 113 ? that makes it properly ordererd at a arithmetic progression series.